.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
The Gitignore file looks ok. However, it would be even better if you add files as .hintrc and others that are not expected to be seen or used after deployment. Take some time and think about what files are not going to be needed after the development stage.